src/utils/components/floatingPosition.test.jsx

```diff
@@ -39,4 +39,12 @@ describe('FloatingPosition', function() {
 	it('matches snapshot', () => {
 		expect(floatingPosition).toMatchSnapshot();
 	});
+
+	it('adjusts popup content position if it overflows the viewport', () => {
```

Comment:
I'd recommend also adding a test to verify `preferredAlignment` is honored if the viewport does **not** clip the content width